package.json

```diff
@@ -4,7 +4,7 @@
   "description": "GraphicaL User Interface for creating and running Scratch 3.0 projects",
   "main": "./src/index.js",
   "scripts": {
-    "build": "npm run clean && webpack --progress --colors --bail",
```

Comment:
Ah, rimraf seems cool. I feel kind of weird about `npm run build` doing a clean first, though. None of our other Scratch 3.0 repositories do that and it's not what I'm used to with other build systems, but maybe it's typical for npm / Node.js / webpack?